dts/bindings/sensor/nxp,imxrt-tmpsns.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Polytech A/S
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP TMPSNS
+
+compatible: "nxp,imxrt-temperature"
```

Comment:
please no soc family name in the compatible , @DerekSnell can you please suggest compatible string